src/block/button/style.js

```diff
@@ -0,0 +1,117 @@
+
+/**
+ * External dependencies
+ */
+import {
+	Advanced,
+	Button,
+	BlockDiv,
+	Typography,
+	Icon,
+	BorderStyle,
+} from '~stackable/block-components'
+import { getUniqueBlockClass } from '~stackable/util'
+import { useDeviceType, useBlockAttributes } from '~stackable/hooks'
+import { useBlockEditContext } from '@wordpress/block-editor'
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment, renderToString } from '@wordpress/element'
+
+export const ButtonStyles = props => {
+	const {
+		...propsToPass
+	} = props
+
+	const deviceType = useDeviceType()
+	const { clientId } = useBlockEditContext()
+	const attributes = useBlockAttributes( clientId )
+
+	propsToPass.blockUniqueClassName = getUniqueBlockClass( attributes.uniqueId )
+	propsToPass.deviceType = deviceType
+	propsToPass.attributes = { ...attributes, clientId }
+	propsToPass.options = {
+		...propsToPass.options,
+		selector: '.stk-button__button',
+		hoverSelector: '.stk-button__button:hover',
+		textSelector: '.stk-button__button > .stk-button__inner-text',
+		textHoverSelector: '.stk-button__button:hover > .stk-button__inner-text',
+	}
+
+	return (
+		<Fragment>
+			<BlockDiv.Style { ...propsToPass } />
+			<Advanced.Style { ...propsToPass } />
+			<Button.Style { ...propsToPass } />
+			<Typography.Style { ...propsToPass } />
+			<Icon.Style
+				{ ...{
+					...propsToPass,
+					options: {
+						...propsToPass.options,
+						selector: '.stk-button__button > .stk-button__svg-wrapper > .stk-button__inner-svg',
+						hoverSelector: '.stk-button__button:hover > .stk-button__svg-wrapper > .stk-button__inner-svg',
+						wrapperSelector: '.stk-button__button',
+					},
+				} } />
```

Comment:
If you adjust `block-components/icon/style.js` and remove `wrapperSelector` this works fine:

```suggestion
			<Icon.Style { ...propsTopass } />
```